doc/classes/KinematicBody2D.xml

```diff
@@ -102,11 +102,11 @@
 			<argument index="5" name="infinite_inertia" type="bool" default="true">
 			</argument>
 			<description>
-				Moves the body along a vector. If the body collides with another, it will slide along the other body rather than stop immediately. If the other body is a [KinematicBody2D] or [RigidBody2D], it will also be affected by the motion of the other body. You can use this to make moving or rotating platforms, or to make nodes push other nodes.
+				Moves the body along a vector. If the body collides with another, it will slide along the other body rather than stop immediately. If the other body is a [KinematicBody2D] or [RigidBody2D], it will also be affected by the motion of the other body. You can use this to either make platforms move or rotate, or to make one node push the other node.
 				This method should be used in [method Node._physics_process] (or in a method called by [method Node._physics_process]), as it uses the physics step's [code]delta[/code] value automatically in calculations. Otherwise, the simulation will run at an incorrect speed.
```

Comment:
I think the current version makes more sense. You can make platforms move and rotate. Using the word either implies it's one or the other not both. Furthermore, you're not limited to one node pushing another node. Each node can push multiple nodes.

Reply:
For line 105, should I change it to "make platforms move and rotate" by "either" erased, or should I revert it back to its previous state as "make moving or rotating platforms"?

Reply:
I think "moving and rotating platforms" is best.